DigitalLearningSolutions.Web/Views/TrackingSystem/Shared/_NavMenuItems.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@using DigitalLearningSolutions.Web.Helpers
+@using Microsoft.Extensions.Configuration
+@inject IConfiguration Configuration
+
+@if (User.HasClaim(c => c.Type == CustomClaimTypes.UserUserAdmin)) {
+  if (User.HasClaim(c => c.Type == CustomClaimTypes.UserCentreAdmin || c.Type == CustomClaimTypes.UserCentreManager)) {
+    <li class="nhsuk-header__navigation-item">
+      <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/tracking/dashboard">
+        Centre
+        <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+          <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+        </svg>

```

Comment:
Not super important, but do you think it might be worth pulling this icon into a partial for better readability (+ it is used in loads of views).

Reply:
Yes, will make a partial for it and refactor the ones in the 2 files I've touched (TrackingSystem navbar and _Layout)